recipes/opencv/4.x/conanfile.py

```diff
@@ -131,6 +133,10 @@ def requirements(self):
             self.requires("quirc/1.1")
         if self.options.get_safe("with_gtk"):
             self.requires("gtk/system")
+        if self.options.dnn:
+            self.requires("protobuf/3.15.5")
+            self.options["protobuf"].PROTOBUF_UPDATE_FILES = True
```

Comment:
`PROTOBUF_UPDATE_FILES`  is an option of OpenCV CMake files, this line doesn't make sense.